### DISCUSSION TITLE
Google Language

### DISCUSSION THREAD

**Discussion by amatya-aditya** [yessGlory17/programmer-browser#47] [Q&A] [answered]
Can I change language in google ? By default, It's showing me the results in regional language. Can I change it to english?

**Comment by yessGlory17** [comment, marked as answer]
Google currently does not consider your browser safe because it is not listed. So I can't login to google. The language setting cannot be changed because it cannot be entered. I'm still working on this issue. I will solve it as soon as possible.

Thanks for your understanding.
